stream-compositions/events/payment-order-events/src/main/resources/events/payment_order.json

```diff
@@ -0,0 +1,230 @@
+    },
+    "CheckImageAvailability": {
+      "type": "string",
+      "example": "AVAILABLE",
+      "enum": [
+        "AVAILABLE",
+        "UNAVAILABLE"
+      ]
+    },
+    "EnrichmentIndicator": {
+      "type": "string",
+      "description": "Indicates whether transaction needs to be enriched or not by following logic.\nSTANDARD_ENRICHMENT: Use default rules in service for enrichment\nDO_ENRICHMENT: Ignore any existing rules and enrich the transaction only if it's not enriched already\nFORCE_ENRICHMENT: Ignore any existing rules and enrich the transaction even if it's already enriched\nNO_ENRICHMENT: Ignore any existing rules and don't attempt to enrich the transaction\nCONFIGURED_ENRICHMENT: Use configured category in service for categorisation of the transaction\n",
+      "example": "DO_ENRICHMENT",
+      "enum": [
+        "STANDARD_ENRICHMENT",
+        "DO_ENRICHMENT",
+        "FORCE_ENRICHMENT",
+        "NO_ENRICHMENT",
+        "CONFIGURED_ENRICHMENT"
+      ]
+    },
+    "Currency": {
+      "title": "Currency",
+      "type": "object",
+      "properties": {
+        "amount": {
+          "description": "The amount in the specified currency",
+          "type": "string",
+          "minimum": -1000000000000000000,
+          "maximum": 1000000000000000000
+        },
+        "currencyCode": {
+          "description": "The alpha-3 code (complying with ISO 4217) of the currency that qualifies the amount",
+          "type": "string",
+          "pattern": "^[A-Z]{3}$"
+        }
+      },
+      "required": [
+        "amount",
+        "currencyCode"
+      ]
+    },
+    "DisputeStatus": {
+      "enum": [
+        "UNDISPUTED",
+        "PENDING",
+        "REJECTED",
+        "REFUNDED"
+      ],
+      "type": "string",
+      "example": "UNDISPUTED"
+    },
+    "TransactionsPostRequestBody": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "arrangementId": {
+          "description": "Reference to the product to which the transaction belongs",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "externalId": {
+          "description": "Internally used unique external identification of the transaction",
+          "maxLength": 300,
+          "type": "string"
+        },
+        "externalArrangementId": {
+          "description": "External reference to the product to which the transaction belongs",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "reference": {
+          "description": "A tag/label issued by the initiator of the transaction in order to be able to refer to the respective transaction",
+          "maxLength": 36,
+          "type": "string"
+        },
+        "description": {
+          "maxLength": 280,
+          "type": "string"
+        },
+        "typeGroup": {
+          "description": "Bank specific code of the group the transaaction type belangs to this to be mapped to in integration",
```

Comment:
transa**a**ction

Reply:
Will remove all payment order event code for now. We'll add it back in a later ticket.